ssh/resource_resource.go

```diff
@@ -50,6 +51,12 @@ func resourceResource() *schema.Resource {
 				Sensitive:    true,
 				RequiredWith: []string{"user"},
 			},
+			"host_private_key": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Sensitive:    true,
+				RequiredWith: []string{"user"},
```

Comment:
Since it's optional, should not use `RequiredWith`

Reply:
Ah! I missed that indeed. I'll fix it & make another commit. Thanks Andy.